packages/test-utils/api-report.md

```diff
@@ -237,6 +240,9 @@ export type TestAppOptions = {
   mountedRoutes?: {
     [path: string]: RouteRef | ExternalRouteRef;
   };
+  icons?: AppIcons;
+  components?: AppComponents;
+  themes?: (Partial<AppTheme> & Omit<AppTheme, 'theme'>)[];
```

Comment:
I'm not sure how changing the `TestAppOptions` relates to the scope of the current Pull Request. To me this feels like a pretty big change to the testing flow. What is your reasoning for this change? Where would this be needed?

Maybe it's easier to create a seperate PR for discussing the `TestAppOptions`. I would suppose that we would definitly want some tests for the new options as well.

Reply:
For snapshot testing it would be nice to get the actual icons, components and themes to the tests.

Reply:
Removed this

Reply:
Yes - I can see the use. A seperate PR might be a bit easier to seperate the discussion on this:)